### PR TITLE
Make it possible to specify filename

### PR DESCRIPTION
**Pull request by okmttdhr**
Support https://github.com/tkalfigo/dotenvenc/issues/2 🙏

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -69,11 +70,12 @@ function decrypt(passwd) {
  * @param     {String}    passwd      the key used to encrypt the .env into .env.enc
  * @returns   {String}                writes the encrypted file to disk at same location where the encrypted file was found and returns its md5 checksum
  */
-function encrypt(passwd) {
+function encrypt(passwd, encryptedFilename) {
```
Comment (tkalfigo) [inline index.js:73]: Variable name is from copy/pasta I guess. We are not encrypting an `encryptedFilename`. Please rename to something more meaningful like `dotEnvFilename` or something less specific.

Also add new `@param` in the function's jsdoc.

File: index.js [hunk 2]
```diff
@@ -50,10 +50,11 @@ function findFileLocation(file) {
  * @param     {String}    passwd      the key used to encrypt the .env into .env.enc we'll use now for decrpypting
  * @returns   {String}                writes the decrypted file to disk at same location where the decrypted file was found and returns its md5 checksum
  */
-function decrypt(passwd) {
+function decrypt(passwd, encryptedFilename) {
```
Comment (tkalfigo) [inline index.js:53]: Please also add new `@param` in the function's jsdoc.

**Comment by tkalfigo** [comment]
Also please bump patch version in package.json

**Comment by tkalfigo** [comment]
Sorry about that. I had to update some deprecated crypto stuff. You'll need to update version based on v2.0.0

Please rebase and fix conflicts and I think it's good to go.

**Comment by timeimp** [comment]
Is this still needing a rebase?

**Comment by tkalfigo** [comment]
Features already in master. See version 2.1.0.

**Pull request closed without merging**